ansible/roles/csm.rr.kyverno_policy/tasks/main.yml

```
@@ -26,13 +26,55 @@
 ---
 - name: Copy the yaml file to the target machine
   copy:
-    src: rrs_topology_policy.yml
-    dest: /tmp/rrs_topology_policy.yml
+    src: "{{ item.src }}"
+    dest: "{{ item.dest }}"
+  loop:
+    - { src: 'cm_static.yml', dest: '/tmp/cm_static.yml' }
+    - { src: 'rrs_topology_policy.yml', dest: '/tmp/rrs_topology_policy.yml' }
```

With the changes made later in the file, you no longer need to copy cm_static, right?

```suggestion
    src: rrs_topology_policy.yml
    dest: /tmp/rrs_topology_policy.yml
```